components/brave_ads/browser/ads_notification_handler.h

```diff
@@ -80,7 +80,10 @@ class AdsNotificationHandler : public NotificationHandler {
    private:
     AdsNotificationHandler* const pointer_;
 
-    DISALLOW_COPY_AND_ASSIGN(UnownedPointer);
+    UnownedPointer(
```

Comment:
it looks like most of these fit into 80 chars, so could you please make these declarations one-liners?
Also they mostly should be present in `public:` section

Reply:
Fixed